docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -397,6 +397,16 @@ https://en.wikipedia.org/wiki/Kurtosis[Quantify] the shape of the distribution o
 include-tagged::{sql-specs}/docs/docs.csv-spec[aggKurtosis]
 --------------------------------------------------
 
+[NOTE]
+====
+`KURTOSIS` cannot be used on top of scalar functions but only directly on a field. So, for example, the following is
+not allowed and an error is returned:
+[source, sql]
+---------------------------------------
+ SELECT KURTOSIS(salary), gender FROM emp GROUP BY gender
```

Comment:
Reading the NOTE I would have expected to see "KURTOSIS ... on top of scalar functions" but I'm seeing KURTOSIS applied on a field.